lib/fastlane/plugin/wpmreleasetoolkit/actions/common/setfrozentag_action.rb

```diff
@@ -27,8 +30,8 @@ def self.run(params)
         end
 
         UI.message("New milestone: #{mile_title}")
-        token = Fastlane::Helper::GithubHelper.github_token
-        Fastlane::Helper::GithubHelper.github_client(token).update_milestone(repository, milestone[:number], title: mile_title)
+
+        github_helper.update_milestone(repository, milestone[:number], title: mile_title)
```

Comment:
Same as above, `update_milestone` is not a method exposed on the `GithubHelper`, but a method on the `Octokit::Client` — hence why previous code was using `…GithubHelper.githubclient.update_milestone`

Similarly, you might want to either expose the `@client` on your `GithubHelper` as an `attr_reader :client` property, to be able to then access it from here like it was with the previous code. Or to create a dedicated trampoline method in `GitHubHelper` to forward the call to the `Octokit::Client`. Up to you what you think is the simpler and best here, I have no strong preference.